website/pages/en/cookbook/enums.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Categorize NFT Marketplaces Using Enums
+---
+
+Learn how to effectively organize information from transactions and marketplace interactions using Enums.
```

Comment:
The first sentence is the most important. I think the benefits should be summarized here so when someone considers reading, they see the end result first. For Enums, I'd say it helps make your code cleaner and less error-prone. I suggest something like: 
"Make your code cleaner and less error-prone by using Enums. Here's a full example of using enums on NFT marketplaces."

Generally, "learn" isn't an outcome people care about so I avoid it.

Reply:
@alinobrasil Great point! This was a strategy I adopted from Vercel, but I like your approach 😉

Reply:
Awesome point!